habitat/tasks/nav/nav.py

```diff
@@ -719,6 +721,60 @@ def update_fog_of_war_mask(self, agent_position):
             )
 
 
+@registry.register_measure
+class DistanceToGoal(Measure):
+    """Distance To Goal metrics
```

Comment:
The docstring here is redundant, name of the class is descriptive of its functionality.